project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,181 @@
+---
+breaks: False
+---
+# Newsletter | 29 January 2020
+
+*Date sent: 01 February 2020*
+
+Hello, 
+
+We're already one month into 2021!
+We hope your year has had a good beginning. 
+For those struggling with these challenging times, we hope that the citizen science community can offer your support and connection. 
+
+Email Georgia at [gaitkenhead@turing.ac.uk](mailto:gaitkenhead@turing.ac.uk) to let us know if we can help you with involvement, and do feel free to share this newsletter to anyone who you think would be interested in joining us. 
+
+They can sign up by following this link: [https://tinyletter.com/AutisticaTuringCitizenScience](https://tinyletter.com/AutisticaTuringCitizenScience).
+
+Best wishes for everyone!
+
+## Project News
+
+### Meet-up sessions
+
+We had our first open online meet-up session last Thursday (25th January). 
+This included a wonderful set of developers, researchers, and autistic collaborators with a whole variety of skills and perspectives. 
+Thank you everyone who took part! 
+
+The next meet-up session will take place on **Thursday 11th February** at an earlier time to accommodate more people. 
```

Comment:
Do you have a sign up or way to add something to a calendar? It's not at all necessary in order to send it out but might be nice to help lower the overhead of adding details and remembering.